drivers/display/Kconfig.st7789h2

```diff
@@ -0,0 +1,44 @@
+# ST7789H2 display driver configuration options
+
+# Copyright (c) 2021, Jeremy LOCHE
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ST7789H2
+	bool "ST7789H2 display driver"
+	depends on GPIO
+	help
+	  Enable driver for ST7789H2 display driver.
+
+choice ST7789H2_PIXEL_FORMAT
+	prompt "Color pixel format"
+	default ST7789H2_RGB565
+	depends on ST7789H2
+	help
+	  Specify the color pixel format for the ST7789H2 display controller.
+
+config ST7789H2_RGB565
+	bool "RGB565"
```

Comment:
Could you please add a `help` and describe there what RGB565 means / refers to.

I see that for now there is only one possible choice value, but as you have already made this a choice, then I assume more options will follow in the future, in which case describing each option carefully is important for other users to understand.

Reply:
Thanks for that comment, yes in fact my implementation support only RGB565 16 bits since I had just the stm32l562e-dk and that's a 16bits bus that's routed on the board. 
Either I do it simpler and remove the "choice" or I could support 12 bits RGB444 fairly easily.

Reply:
Fixed this by removing this config prompt and providing support to RG565 only. RGB444 is not available through the display device API so the only meaningful ones would have been RGB888 24 bits instead but the ST7789 only supports up to 18bits pixel data so I didn't feel like spending time on this 🙃